.github/workflows/package_all.yaml

```diff
@@ -89,7 +96,7 @@ jobs:
       - name: "Build package"
         run: |
           cd release/package
-          ./run.sh package debian-11 --for-docker
+          ./run.sh package debian-11 ${{ github.event.inputs.build_type }} --for-docker
           ./run.sh docker
```

Comment:
@gitbuda 
I don't think that we need to add build type for `run.sh docker` because it will use package from debian-11 from line 99.
The only thing that inside, docker script as temporary directory uses release folder here: https://github.com/memgraph/memgraph/blob/build-release-bins-with-debug-info/release/package/run.sh#L138
I'm not sure that we need to change this.